packages/oas/test/operation/lib/get-requestbody-examples.test.ts

```diff
@@ -5,20 +5,54 @@ import cleanStringify from '../../__fixtures__/json-stringify-clean.js';
 
 let operationExamples: Oas;
 let petstore: Oas;
+let webhooksOas: Oas;
 
 beforeAll(async () => {
   operationExamples = await import('../../__datasets__/operation-examples.json').then(r => r.default).then(Oas.init);
   await operationExamples.dereference();
 
   petstore = await import('@readme/oas-examples/3.0/json/petstore.json').then(r => r.default).then(Oas.init);
   await petstore.dereference();
+
+  webhooksOas = await import('@readme/oas-examples/3.1/json/webhooks.json').then(r => r.default).then(Oas.init);
 });
 
 test('should return early if there is no request body', () => {
   const operation = operationExamples.operation('/nothing', 'get');
   expect(operation.getRequestBodyExamples()).toStrictEqual([]);
 });
 
+test('should generate a request example for an operation with an undefined example value', async () => {
```

Comment:
now that we understand the underlying issue a bit better, i think this would be a better title:

```suggestion
test('should re-intialize the request examples after the oas is dereferenced', async () => {
```